### PR TITLE
V1.5

### PR DESCRIPTION
**Pull request by rwarren2**
A first batch of changes, @Stephen-Gates do you want to give this a quick look over? I'm going to work on documentation for this next.

### CODE REVIEW

File: documentation/terms/Dead_people_place.en [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
+A rather poorly named class, this is the super-class of all other classes. It
+is not meant to be instantiated but groups the subclasses together in a
+common purpose.
+
+This should be the reverse of a <a href="http://www.geonames.org/ontology#P.PPL">populated place</a> according to the <a href="http://www.geonames.org/ontology/ontology_v2.2.1.rdf">Geonames ontology</a>.
```
Comment (Stephen-Gates) [inline documentation/terms/Dead_people_place.en:6]: Latest version of genomes is v3.2

File: ontology/graves.owl [hunk 2]
```diff
@@ -1,67 +1,319 @@
...
+   <dcterms:description xml:lang="fr">L'ontologie Muninn Graves se concerne avec les restes humains, les tombes et les lieux où ils se trouvent ainsi que les monuments qui commémorent des personnes ou des événements.</dcterms:description>
+   <dcterms:description xml:lang="de">Die Muninn Graves Ontology befasst sich mit menschlichen Überresten, Gräbern und Orten, an denen sie gefunden wurden, und Denkmälern, die an Personen oder Ereignisse erinnern.</dcterms:description>
+   <owl:priorVersion rdf:resource="https://rdf.muninn-project.org/ontologies/graves-20121031"/>
+   <owl:versionInfo rdf:datatype="http://www.w3.org/2001/XMLSchema#decimal">ONTOLOGY_VERSION</owl:versionInfo>
    <vs:term_status>Stable</vs:term_status>
-   <dc:date rdf:datatype="http://www.w3.org/2001/XMLSchema#date">20121031</dc:date>
+   <dcterms:date rdf:datatype="http://www.w3.org/2001/XMLSchema#date">ONTOLOGY_DATE</dcterms:date>
+   <void:inDataset rdf:resource="#datasetdefinition"/>
+   <foaf:logo rdf:resource="http://rdf.muninn-project.org/ontologies/UnknownGrave.png"/>
+   <cc:license rdf:resource="http://creativecommons.org/licenses/by/3.0/"/>
```
Comment (Stephen-Gates) [inline ontology/graves.owl]: see https://github.com/muninn/graves/issues/42

File: ontology/graves.owl [hunk 3]
```diff
@@ -1,67 +1,319 @@
...
+   <dcterms:description xml:lang="de">Die Muninn Graves Ontology befasst sich mit menschlichen Überresten, Gräbern und Orten, an denen sie gefunden wurden, und Denkmälern, die an Personen oder Ereignisse erinnern.</dcterms:description>
+   <schema:description xml:lang="en">The Muninn Graves Ontology deals with human remains, graves and places they are found, and monuments that commemorate people or events.</schema:description>
+   <schema:description xml:lang="fr">L'ontologie Muninn Graves se concerne avec les restes humains, les tombes et les lieux où ils se trouvent ainsi que les monuments qui commémorent des personnes ou des événements.</schema:description>
+   <schema:description xml:lang="de">Die Muninn Graves Ontology befasst sich mit menschlichen Überresten, Gräbern und Orten, an denen sie gefunden wurden, und Denkmälern, die an Personen oder Ereignisse erinnern.</schema:description>
+   <cc:license rdf:resource="http://creativecommons.org/licenses/by/3.0/"/>
```
Comment (Stephen-Gates) [inline ontology/graves.owl]: see https://github.com/muninn/graves/issues/42

File: ontology/graves.owl [hunk 4]
```diff
@@ -898,204 +1461,465 @@
...
+<!-- This section according to time/prov -->
+<rdf:Description rdf:about="http://www.w3.org/ns/prov#Activity">
+ <rdfs:comment xml:lang="en">According to https://www.w3.org/TR/owl-time/#time-prov</rdfs:comment>
+ <rdfs:comment xml:lang="en">D'après https://www.w3.org/TR/owl-time/#time-prov</rdfs:comment>
+ <rdfs:subClassOf rdf:resource="http://www.w3.org/2006/time#TemporalEntity"/>
+</rdf:Description>
+
+<rdf:Description rdf:about="http://www.w3.org/ns/prov#InstantaneousEvent">
+ <rdfs:comment xml:lang="en">According to https://www.w3.org/TR/owl-time/#time-prov</rdfs:comment>
+ <rdfs:comment xml:lang="en">D'après https://www.w3.org/TR/owl-time/#time-prov</rdfs:comment>
```
Comment (Stephen-Gates) [inline ontology/graves.owl]: should be lang="fr"

File: ontology/graves.owl [hunk 5]
```diff
@@ -898,204 +1461,465 @@
...
+ <rdfs:subClassOf rdf:resource="http://www.w3.org/2006/time#Instant"/>
+</rdf:Description>
+
+<rdf:Description rdf:about="http://www.w3.org/ns/prov#atTime">
+ <rdfs:comment xml:lang="en">According to https://www.w3.org/TR/owl-time/#time-prov</rdfs:comment>
+ <rdfs:comment xml:lang="en">D'après https://www.w3.org/TR/owl-time/#time-prov</rdfs:comment>
```
Comment (Stephen-Gates) [inline ontology/graves.owl]: lang="fr"

File: ontology/graves.owl [hunk 6]
```diff
@@ -898,204 +1461,465 @@
...
+ <rdfs:subPropertyOf rdf:resource="http://www.w3.org/2006/time#inXSDDateTime"/>
+</rdf:Description>
+
+<rdf:Description rdf:about="http://www.w3.org/ns/prov#endedAtTime">
+ <rdfs:comment xml:lang="en">According to https://www.w3.org/TR/owl-time/#time-prov</rdfs:comment>
+ <rdfs:comment xml:lang="en">D'après https://www.w3.org/TR/owl-time/#time-prov</rdfs:comment>
```
Comment (Stephen-Gates) [inline ontology/graves.owl]: lang="fr"

File: documentation/terms/Dead_people_place.en (hunk 1 above)
Comment (rwarren2) [reply documentation/terms/Dead_people_place.en:6]: Going to update the documentation in the next PR.

File: ontology/graves.owl [hunk 7]
```diff
@@ -898,204 +1461,465 @@
...
+ <owl:propertyChainAxiom rdf:parseType="Collection">
+  <owl:ObjectProperty rdf:about="http://www.w3.org/2006/time#hasEnd"/>
+  <owl:ObjectProperty rdf:about="http://www.w3.org/2006/time#inXSDDateTime"/>
+ </owl:propertyChainAxiom>
+</rdf:Description>
+
+<rdf:Description rdf:about="http://www.w3.org/ns/prov#startedAtTime">
+ <rdfs:comment xml:lang="en">According to https://www.w3.org/TR/owl-time/#time-prov</rdfs:comment>
+ <rdfs:comment xml:lang="en">D'après https://www.w3.org/TR/owl-time/#time-prov</rdfs:comment>
```
Comment (Stephen-Gates) [inline ontology/graves.owl]: lang="fr"

**Review by Stephen-Gates — COMMENTED** [review]
Added some single line comments to graves.owl. I'm wondering if every Property needs a description? In some cases there's no description for a property or its inverse, e.g. `containsGrave`. Some Classes don't have a description, e.g. `Ossuary`

**Comment by rwarren2** [comment]
Yes, all properties and class need document. I have a lineup of updates coming for that also.

**Comment by rwarren2** [comment]
CI Test entry.

File: documentation/graves-template-en.html [hunk 8]
```diff
@@ -0,0 +1,620 @@
...
+    <script id="vocab" type="application/ld+json">
+     m4_include(graves-ONTOLOGY_DATE.jsonld)
+    </script>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
+    <script src="js/exampleSwitcher.js"></script>  
+  </head>
+
+  <body>
+  {_header_}
+    <p class="copyright"><a rel="license" href="http://creativecommons.org/licenses/by-sa/3.0/">
```
Comment (Stephen-Gates) [inline documentation/graves-template-en.html:24]: Change to Creative Commons BY SA v4.0?

File: documentation/graves-template-en.html [hunk 9]
```diff
@@ -0,0 +1,620 @@
...
+      href="#Container_for_human_remains">receptacle</a> <a
+      href="#Remains">for human
+      remains</a>. Subclasses are
+      used for cases of <a hreg="#Empty_grave">Empty Graves</a> or <a
+      hreg="#Mass_grave">Mass Graves</a>. 
+    </p>
+    <p><a href="#Remains">Human remains</a> are classes which deal with
+    <a href="#Skeleton">full skeletons</a>, the <a href="#Partial_remains">partial remains</a> of a human being (especially in the
+    case of some arch digs) and their <a href="#Ashes">ashes</a> after
+    creation.</p>
```
Comment (Stephen-Gates) [inline documentation/graves-template-en.html:197]: Cremation

File: documentation/graves-template-en.html [hunk 10]
```diff
@@ -0,0 +1,620 @@
...
+ 
+    <p></p>
+
+    <!-- The following is the script-generated list of classes and properties -->
+    {_terms_}
+    <!-- End of the terms list -->
+
+    <h2 id="#conclusion">8.Conclusion and Future Work</h2>
+    
+    <p>The graves ontology is an initial attempt at documenting graves and
```
Comment (Stephen-Gates) [inline documentation/graves-template-en.html:542]: Do you want to mention Github issues and discussion forum?

File: documentation/graves-template-en.html [hunk 11]
```diff
@@ -0,0 +1,620 @@
...
+    <ul>
+    <li>1.5</li>
+    <ul>
+    <li> - Major addition to ontology / dataset header for schema.org and dcat.</li>
+    <li> - Ontology now listed by zenodo doi.</li>
+    <li> - Change of rdfs:comment to skos:definition for term documentation.</li>
+    <li> - Multi-serialization examples.</li>
+    <li> - json-ld serialization available by content negotiation.</li>
+    <li> - French documentation now available.</li>
+    <li> - Many fixed to broken examples.</li>
```
Comment (Stephen-Gates) [inline documentation/graves-template-en.html:561]: Many fixes...?

File: documentation/graves-template-en.html [hunk 12]
```diff
@@ -0,0 +1,620 @@
...
+    <li>    - Fixed inverse properties.</li>
+    </ul>
+    <li>0.1 - Initial unstable release.</li>
+    </ul>
+    </ul>
+
+    <h2 id="TODO">TODO</h2>
+       <ul>
+       <li>Explain ontology consistency checking.</li>
+       <li>Add funeral urns, storage boxes and body bags?</li>
```
Comment (Stephen-Gates) [inline documentation/graves-template-en.html:606]: Funeral urns - done? Or do you just need to add an example?

**Comment by rwarren2** [comment]
@Stephen-Gates Going to merge this to get the ontology fixes but the documentation isn't there yet.

**Pull request merged**